GetStream/stream-chat-android
# [3449] Create script to run emulator locally

### 🎯 Goal

Fixes #3449 

Using CI for snapshot tests is slow and not always convenient. We need local scripts to run our snapshot tests, which produce consistent results.

### 🧪 Testing

1. Run the script to create emulator:
```
./scripts/emulator_create.sh
```

2. Run the script to start emulator:
```
./scripts/emulator_start.sh
```

3. Wait for the emulator to start and run snapshot tests:
```
./gradlew stream-chat-android-ui-uitests:executeScreenshotTests -Pandroid.testInstrumentationRunnerArguments.filter=io.getstream.chat.android.uitests.util.SnapshotTestFilter 
```

4. Ensure that snapshot tests are passing by reviewing the report:
```
%YOUR REPO FOLDER%/stream-chat-android/stream-chat-android-ui-uitests/build/reports/shot/debug/verification/index.html
```

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
